models/public/index.md

```diff
@@ -77,6 +77,7 @@ detect areas with complex shape.
 | Model Name                | Implementation | OMZ Model Name | Accuracy | GFlops | mParams |
 | ------------------------- | -------------- | -------------- | -------- | ------ | ------- |
 | DeepLab V3                | TensorFlow\*   | [deeplabv3](./deeplabv3/deeplabv3.md) | 66.85% | 11.469 | 23.819 |
+| Conv-Deconvnet            | PyTorch\*      | [conv-deconvnet](./conv-deconvnet/conv-deconvnet.md) | 82.7% | 141.62 | 84.03 |
```

Comment:
Please rename `Conv-Deconvnet`

Reply:
Why in model.yml, `model_type: classification`, but in index.md, it is located in semantic segmentation section?

Reply:
Looks like it's segmentation, yes.

Reply:
@alexeyhorkin, friendly reminder